### ISSUE TITLE
SIGSEGV when trying to update mods

### ISSUE THREAD

**Issue by SandaruKasa** [PrismLauncher/PrismLauncher#893]
### Operating System

Linux

### Version of Prism Launcher

PrismLauncher 6.3
39bba9cbfadc0aac3ef571a80e78a9fa3f5a0028

`prismlauncher` AUR package. Built with LTO, here's a small extract from my `makepkg.conf`:
```ini
CFLAGS="-march=native -mtune=native -O2 -pipe -fno-plt -fexceptions \
        -Wp,-D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security \
        -fstack-clash-protection -fcf-protection"
CXXFLAGS="$CFLAGS -Wp,-D_GLIBCXX_ASSERTIONS"
LDFLAGS="-Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now"
LTOFLAGS="-flto=auto"
```

### Version of Qt

Qt 6.4.2

### Description of bug

Trying to update mods causes a SIGSEGV:
[PrismLauncher-0.log](https://github.com/PrismLauncher/PrismLauncher/files/10854107/PrismLauncher-0.log)

### Steps to reproduce

* Open the instance edit menu, mod tab
* Press `Check for Updates`
* This windows pops up (because I have a replaymod-patched version of sodium installed from the replaymod's site):
![image](https://user-images.githubusercontent.com/50824690/221960464-a59a6cba-54f2-41b6-bd6e-9bca5553c253.png)
* Select Skip
* It crashes

Alternatively, select `Confirm`, wait for it to fail to find metadata, click `Ok`. It crashes.

### Suspected cause

Not `Sodium` apparently. Removing both versions of it before trying to update did not make the crash go away.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by SandaruKasa** [comment]
Probably a duplicate of #659 & #875 (now we have all: MacOS, Windows, and Linux reported, heh). Somehow github search didn't bring those two up for me. Or maybe I just can't read.

However, #802 did not fix the issue for me: I've installed `prismlauncher-git`, it gave me a "PrismLauncher 7.0-develop 8ce8055aeb3e610088ff5a08f509862c1feafd8f" version, which is _after_ #802 has been merged in d2fd0fd8e896e3d5dc0449ae719fdaa7d27968a5, but the issue still persists for me.

**Comment by SandaruKasa** [comment]
I've just realized the instance I was trying to update the mods for wasn't a proper instance, but just a bunch of leftover files. [Oops](https://youtu.be/SYCP71qcYZw).
![image](https://user-images.githubusercontent.com/50824690/221968621-db77e2e9-6d73-4a9c-84aa-6c1580032b10.png)
![image](https://user-images.githubusercontent.com/50824690/221968691-8fb1a33e-dea0-402d-9a69-aab4cd923d35.png)
So I guess an error is to be expected in this case. IDK if it would be worth the effort to try to make the launcher give a graceful error here instead of crashing.

Leaving the issue open for now, but feel free to close it if this is a wontfix.
